data/routes/projectRoutes.js

```diff
@@ -0,0 +1,47 @@
+
+const router = require('express').Router();
+
+const knex = require('knex');
+const knexConfig = require('../../knexfile');
+
+const db = knex(knexConfig.development);
+
+
+// router.use(express.json());
+
+router.get('/', async (req, res) => {
+    try {
+        const showProjects = await db('projects')
+        res.status(200).json(showProjects)
+    } catch (err){
+        res.status(500).json(err)
+    }
+});
+
+router.get('/:id', async (req, res) => {
+    try{
+        const project = await db('projects')
+        // .select('actions.id', 'actions.action_description').from('projects')
+        .select('*').from('projects')
+        .innerJoin('actions', 'projects.id', 'actions.project_id')
+        .where({'project_id': req.params.id})
+        res.status(200).json(project)
```

Comment:
I see what you were trying to do here but SQL doesn't return data in the structure required by the project guidlines. You can do it by first getting the project object, then getting the array of actions by project id and then appending the array of actions to the project object.